llvm/test/Transforms/Inline/update_invoke_value_profile.ll

```diff
@@ -0,0 +1,54 @@
+; A pre-commit test to show that value profiles associated with invoke are not updated.
```

Comment:
Can the two test cases be merged into one? i.e.,  callee makes calls to inner_callee and *func .

Reply:
done.